test/integration/pnpm-support/test/index.test.js

```diff
@@ -140,6 +140,48 @@ describe('pnpm support', () => {
     })
   })
 
+  it('pnpm works with outputStandalone', async () => {
```

Comment:
It seems these checks pass even without the changes in this PR, can we track down the test conditions in the related issues so we ensure we don't regress?

Reply:
I replaced the test with an browser integration test